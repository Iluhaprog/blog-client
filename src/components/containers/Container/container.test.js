import React from 'react';
import { render } from '@testing-library/react';
import { Container } from '.';

test('should render container', () => {
    const { asFragment } = render(<Container></Container>);
    expect(asFragment()).toMatchSnapshot();
});