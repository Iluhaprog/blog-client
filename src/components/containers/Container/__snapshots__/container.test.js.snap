// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render container 1`] = `
<DocumentFragment>
  <div
    class="container"
  />
</DocumentFragment>
`;
